homeassistant/components/permobil/sensor.py

```diff
@@ -0,0 +1,215 @@
+        device_class=SensorDeviceClass.DURATION,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    PermobilSensorEntityDescription(
+        # Watt hours the battery can store given battery health
+        value_fn=lambda data: data.battery[BATTERY_MAX_AMPERE_HOURS]
+        * BATTERY_ASSUMED_VOLTAGE,
+        key=BATTERY_MAX_AMPERE_HOURS,
+        translation_key="max_watt_hours",
+        native_unit_of_measurement=UnitOfEnergy.WATT_HOUR,
+        device_class=SensorDeviceClass.ENERGY_STORAGE,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    PermobilSensorEntityDescription(
+        # Current amount of watt hours in battery
+        value_fn=lambda data: data.battery[BATTERY_AMPERE_HOURS_LEFT]
+        * BATTERY_ASSUMED_VOLTAGE,
+        key=BATTERY_AMPERE_HOURS_LEFT,
+        translation_key="watt_hours_left",
+        native_unit_of_measurement=UnitOfEnergy.WATT_HOUR,
+        device_class=SensorDeviceClass.ENERGY_STORAGE,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    PermobilSensorEntityDescription(
+        # Distance that can be traveled with full charge given battery health (km)
+        value_fn=lambda data: data.battery[BATTERY_MAX_DISTANCE_LEFT],
+        key=BATTERY_MAX_DISTANCE_LEFT,
+        translation_key="max_distance_left",
+        native_unit_of_measurement=UnitOfLength.KILOMETERS,
+        device_class=SensorDeviceClass.DISTANCE,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    PermobilSensorEntityDescription(
+        # Distance traveled today monotonically increasing, resets every 24h (km)
+        value_fn=lambda data: data.daily_usage[USAGE_DISTANCE],
+        key=USAGE_DISTANCE,
+        translation_key="usage_distance",
+        native_unit_of_measurement=UnitOfLength.KILOMETERS,
+        device_class=SensorDeviceClass.DISTANCE,
+        state_class=SensorStateClass.TOTAL_INCREASING,
+    ),
+    PermobilSensorEntityDescription(
+        # Number of adjustments monotonically increasing, resets every 24h
+        value_fn=lambda data: data.daily_usage[USAGE_ADJUSTMENTS],
+        key=USAGE_ADJUSTMENTS,
+        translation_key="usage_adjustments",
+        native_unit_of_measurement="adjustments",
+        state_class=SensorStateClass.TOTAL_INCREASING,
+    ),
+    PermobilSensorEntityDescription(
+        # Largest number of adjustemnts in a single 24h period, never resets
+        value_fn=lambda data: data.records[RECORDS_SEATING],
+        key=RECORDS_SEATING,
+        translation_key="record_adjustments",
+        native_unit_of_measurement="adjustments",
+        state_class=SensorStateClass.TOTAL_INCREASING,
+    ),
+)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: config_entries.ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Create sensors from a config entry created in the integrations UI."""
+
+    # load the coordinator from the config
```

Comment:
Most of these comments both here and in other files are stating the obvious so I would recommend you to clean it out unless they actually explain something